bin/codemods/src/config.js

```diff
@@ -62,4 +62,5 @@ const codemodArgs = {
 module.exports = {
 	codemodArgs,
 	jscodeshiftArgs,
+	recastOptions,
```

Comment:
Actually none of the codemods currently receive this (e.g. [sort-imports codemod](https://github.com/Automattic/wp-calypso/blob/master/bin/codemods/src/sort-imports.js#L91).

How would I know if any of those codemods are active anymore?

Reply:
> How would I know if any of those codemods are "active" anymore? I feel that if everyone has been using them without this configuration, I would need to test them all if I change this.

I _think_ we've run most of them by now; but some of them just not yet on the entire codebase, e.g. `sort-imports`. (I don't think you need to test them all at this point.)